conda-forge.yml

```diff
@@ -1,8 +1,7 @@
 azure:
-  settings_win:
+  settings_osx:
     pool:
-      vmImage: windows-2019
-  store_build_artifacts: true
+      vmImage: macos-12
```

Comment:
Thank you for pointing this out. I think maybe we can set `MKS_FOUND=0` or something like that. I'm going to let these builds run since I think linux should now pass. I think we had been disabling MKLDNN for a while now.... unintentionally.

We will have to start to build with macos-12 https://github.com/conda-forge/conda-forge.github.io/issues/1798#issuecomment-1186162832= anyway....

Reply:
I think once i fixed the string parsing it claims to follow the flag:


```
  --   USE_PYTORCH_METAL_EXPORT     : OFF
  --   USE_MPS               : OFF
  --   USE_FFTW              : OFF
  --   USE_MKL               :
  --   USE_MKLDNN            : 0
  --   USE_NCCL              : OFF
  --   USE_NNPACK            : ON
  --   USE_NUMPY             : ON
```

Reply:
>  I didn't think of that before

You did think of that. You just had your code two lines too early.

>  The main section said you built it locally for linux cpu? Did the find_path thing fix it? Wow!!!

Yeah, builds are quicker and I think it worked trough the docker process.